README.rst

```diff
@@ -45,7 +46,19 @@ The quick way::
 For more details see the install section in the documentation:
 https://spidermon.readthedocs.io/en/latest/installation.html
 
+For implementing any of Spidermon's amazing features such as `actions <https://spidermon.readthedocs.io/en/latest/actions.html>`_ or `item validations <https://spidermon.readthedocs.io/en/latest/item-validation.html>`_. We recommend to check the documentation for instructions to download specific packages needed for the, to work.  
+
 Documentation
 =============
 
 Documentation is available online at https://spidermon.readthedocs.io/ and in the ``docs`` directory.
+
+Contributing 
+============
+
+Looking forward to improve Spidermon, checkout out our awesome `CONTRIBUTING.md <https://github.com/scrapinghub/spidermon/blob/master/CONTRIBUTING.rst>`_ to get you started.
+
+License
+=======
+
+All source code is under `BSD License <https://github.com/scrapinghub/spidermon/blob/master/LICENSE>`_ 
```

Comment:
Linking to the master version of the LICENSE file might not be a good idea. If the license ever changes, old versions would be pointing to the wrong license.

If GitHub can handle it well on the README rendering, I think we should add ‘BSD-license’ to the description of Spidermon, and link that text to the actual LICENSE file using [`:doc:`, `:download:` or similar](http://www.sphinx-doc.org/es/stable/markup/inline.html#cross-referencing-documents).

Reply:
Thanks, didn't quite get that. I think I may not be completely acquainted with the ScrapingHub workflow, but how would old versions point to the wrong license?
- If the license ever changes, the file would be named `LICENSE` only
- Wouldn't the master branch hold the latest version of the LICENSE we are choosing? 

>  I think we should add ‘BSD-license’ to the description of Spidermon, and link that text to the actual LICENSE file using :doc:, :download: or similar.

There is already a badge for it above. I am not sure adding it to the description would do much good. (To be sure, where exactly is `description of spidermon` - They are in many places)

Reply:
Regarding pointing to the wrong license, picture this scenario:

#. Spidermon 1.0 includes a README like the one you suggest.
#. Spidermon 2.0 changes the license.

If someone downloads the source code of 1.0, there is a LICENSE file with one license, and a README file that points to a different, online LICENSE file with a different license. I think that would be a potential legal issue.

As for the BSD license, I did not notice the badge; just ignore my comment.

Reply:
I never considered downloading into the scenario, this really opens up a new perspective for me. Thanks. Will remove it.